pkg/execution/execution_test.go

```diff
@@ -2170,7 +2169,7 @@ func TestExecutor_HTTPJSONDataSourceWithBody(t *testing.T) {
 		restServer := createRESTServer(wantString)
 		defer restServer.Client()
 
-		run(t, restServer, `{{ .arguments.input.query }}`, `{"query": { "meta_data": { "test": "{foo: \"bar\"}" } }`, expectedResult)
+		run(t, restServer, `{{ .arguments.input.query }}`, `{"query": { "meta_data": { "test": "{\"foo\": \"bar\", \"re\":\"\\w+\"}" } }`, expectedResult)
```

Comment:
Can you please keep the existing test case and add a new one?

Reply:
I think it is totally valid to have the same case here.